portfolio/src/main/webapp/index.html

```diff
@@ -207,7 +207,7 @@ <h4>my relevant coursework</h4>
 
         <section id="comments">
           <h3>arrington's friends comments</h3>
-          <form action="/data" method="POST">
+          <form action="/data?pag=0" method="POST">
```

Comment:
I thought changing the numComments was moved to a different servlet so the post of /data would be creating new comments.

Thinking about it again, you might not even need to do this server side if you store this on the client in sessionStorage and send this and the page number as part of the get request.